apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2QSchemaUserTypeIT.java

```diff
@@ -237,6 +237,39 @@ public void udtUpdateComplex() {
     assertThat(fieldNames).isEqualTo(Arrays.asList("age", "firstname"));
   }
 
+  // Externally reported issued
```

Comment:
is this a small typo? meant to say `issue`?

Reply:
typo, thanks!